content/docs/command-reference/run.md

```diff
@@ -218,6 +230,28 @@ $ dvc run -d parsingxml.R -d Posts.xml \
           Rscript parsingxml.R Posts.xml Posts.csv
 ```
 
+Use a subset of hyperparameters from the default params file `params.yaml`. The
+parameters should be readed from user's code. DVC can granulary track
+dependencies for the defined subset of parameters for `dvc repro`:
```

Comment:
Didn't quite understand this paragraph.
Also, it should probably have a separate example heading even if it's a continuation of the basic example above.